jamjar/jamjar/videos/views.py

```diff
@@ -177,3 +224,26 @@ def delete(self, request, id):
         self.video.delete()
         return self.success_response("Video with id {} successfully deleted.".format(id))
 
+class VideoWatchView(BaseView):
+    model = Video
+
+    """
+    Description:
+        Given a video id, incremement that video count.  We want to make this
+        endpoint as cheap as possible, so we do some funky stuff here.
+         - We don't authenticate
+         - We use an F expression to both find and update the row in the DB
+           at the same time
+
+    Request:
+        GET /videos/:video_id/watching/
+
+    Response:
+        True
+    """
+    # Don't authenticate this
+    #@authenticate
+    def get(self, request, id):
```

Comment:
@markkohdev i think this should be a POST or a PUT -- kind of weird that a GET will change db state :)